*** Exception: foo
CallStack (from ImplicitParams):
  error, called at <interactive>:2:1 in interactive:Ghci1
Stopped at <exception thrown>
_exception :: e = _
Stopped at <exception thrown>
_exception :: e = _
-1  : main (../Test7.hs:2:18-28)
-2  : main (../Test7.hs:2:8-29)
<end of history>
Logged breakpoint at ../Test7.hs:2:18-28
_result :: a14
Logged breakpoint at ../Test7.hs:2:8-29
_result :: IO a14
no more logged breakpoints
Logged breakpoint at ../Test7.hs:2:18-28
_result :: a14
Stopped at <exception thrown>
_exception :: e
already at the beginning of the history
_exception = SomeException
               (ErrorCallWithLocation
                  "foo"
                  "CallStack (from ImplicitParams):
  error, called at ../Test7.hs:2:18 in main:Main")
_result :: a14 = _
_exception :: SomeException = SomeException
                                (ErrorCallWithLocation
                                   "foo"
                                   "CallStack (from ImplicitParams):
  error, called at ../Test7.hs:2:18 in main:Main")
*** Exception: foo
CallStack (from ImplicitParams):
  error, called at ../Test7.hs:2:18 in main:Main
*** Exception: foo
CallStack (from ImplicitParams):
  error, called at ../Test7.hs:2:18 in main:Main
